tests/python/unittest/test_arith_intset.py

```diff
@@ -59,14 +59,37 @@ def test_check():
 
     # multiple compare operators
     res2 = tvm.arith.DeduceBound(a, (a+b>3)>c , {b: b_s, c: c_s}, {})
-    assert res1.is_nothing()
+    assert res2.is_nothing()
 
     # multiple target variable
     res2 = tvm.arith.DeduceBound(a, a*2-a>b, {b: b_s}, {})
-    assert res1.is_nothing()
+    assert res2.is_nothing()
+
+def test_deduce_relax():
+    def test(a1, a2):
+        a = tvm.var('a')
+        b = tvm.var('b')
+        b_s = tvm.arith.intset_interval(a1, a2)
+        e0 = b + a*4
+        res1 = tvm.arith.DeduceBound(a, e0<17, {b: b_s}, {b: b_s})
+        assert tvm.ir_pass.Simplify((res1.max() * 4 + b_s.max()) < 17).value == 1
+
+        res1 = tvm.arith.DeduceBound(a, e0>17, {b: b_s}, {b: b_s})
+        assert (tvm.ir_pass.Simplify((res1.min() * 4 + b_s.min()) > 17)).value == 1
+
+        res1 = tvm.arith.DeduceBound(a, e0<=17, {b: b_s}, {b: b_s})
+        assert (tvm.ir_pass.Simplify((res1.max() * 4 + b_s.max()) <= 17)).value == 1
+      
+        res1 = tvm.arith.DeduceBound(a, e0>=17, {b: b_s}, {b: b_s})
+        assert (tvm.ir_pass.Simplify((res1.min() * 4 + b_s.min()) >= 17)).value == 1
+       
+    test(0, 4)
+    test(1, 5)
+    test(2, 6)
```

Comment:
for better coverage, we need to be able to give case when the co-efficient is negative

Reply:
add a testcase with more complicated nest

Reply:
do you mean e0 = b + a*(-4)?